libnmstate/nm/connection.py

```diff
@@ -387,20 +387,26 @@ def _delete_connection_callback(self, src_object, result, user_data):
     @staticmethod
     def _commit_changes_callback(src_object, result, user_data):
         mainloop, nmdev = user_data
```

Comment:
Proposal for the commit message:

Handle `nmdev` being `None` by getting the device name from `src_object` for error logging. This fixes the following traceback:

```
  File "/usr/lib/python3.6/site-packages/libnmstate/nm/connection.py", line 398, in _commit_changes_callback
    e, nmdev.props.interface, nmdev.props.state
AttributeError: 'NoneType' object has no attribute 'props'
```

Reply:
Too much details for a commit message IMO, but if you prefer it, I will not argue.
Thanks!

Reply:
I like the details :-)